src/types/FullTodo.ts

```diff
@@ -0,0 +1,6 @@
+import { Todo } from './Todo';
+import { User } from './User';
+
+export interface FullTodo extends Todo {
+  user: User | null;
```

Comment:
Consider using a more descriptive name for the property `user`. For example, `assignedUser` or `assignedTo`. 🤔